packages/frontend/src/pages/ProjectSettings.styles.tsx

```diff
@@ -21,3 +21,5 @@ export const Title = styled(H3)<{ marginBottom?: boolean }>`
             margin: 0 0 20px;
         `}
 `;
+
+export const ProjectAccessWrapper = styled.div``;
```

Comment:
Was this intentional?

Reply:
Yes, I created the div style but I didn't give it any properties in the end, should I remove it ?